src/java.base/share/classes/java/io/Closeable.java

```diff
@@ -50,4 +52,35 @@ public interface Closeable extends AutoCloseable {
      * @throws IOException if an I/O error occurs
      */
     public void close() throws IOException;
+
+    /**
+     * Closes this stream and releases any system resources associated
+     * with it. If the stream is already closed then invoking this
+     * method has no effect.
+     *
+     * @implSpec
+     * The default implementation is equivalent for this stream to:
+     * {@snippet lang=java :
+     * try {
+     *     close();
+     * } catch (IOException cause) {
+     *     throw new UncheckedIOException(cause);
+     * }
+     * }
+     * Therefore if an {@linkplain UncheckedIOException} is thrown then its
+     * {@linkplain UncheckedIOException#getCause() getCause} method will return the
+     * {@linkplain IOException} thrown by {@link #close() close}.
```

Comment:
These links should all be `@link` as they should be rendered like code like the `{@code IOException}` below.

Reply:
I agree. So changed in c89945b81a9d8312087593ed5e6ccba06a413ce0.